src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -252,30 +136,29 @@ const GuideStore = Reflux.createStore({
       });
 
     if (!this.state.forceShow) {
-      guideKeys = guideKeys.filter(key => !this.state.guides[key].seen);
+      const user = ConfigStore.get('user');
+      guideKeys = guideKeys.filter(
+        key =>
+          !this.state.guides[key].seen &&
+          (user.isSuperuser || new Date(user.dateJoined) > new Date(2019, 6, 1))
```

Comment:
Could we add a comment to explicitly call out that we don't want to show it to members before we implemented the new guide functionality.